tests/fast/BlobGranuleCorrectness.toml

```diff
@@ -8,3 +8,27 @@ testTitle = 'BlobGranuleCorrectnessTest'
     [[test.workload]]
     testName = 'BlobGranuleVerifier'
     testDuration = 120.0
+
+    [[test.workload]]
+    testName = 'RandomClogging'
+    testDuration = 120.0 
+
+    [[test.workload]]
+    testName = 'Rollback'
+    meanDelay = 30.0
```

Comment:
Might as well add the non-clean version of BlobGranuleCorrectnessLarge while you're at it to complete the set of tests?